src/MoBi.Presentation/Presenter/Main/ModuleExplorerPresenter.cs

```diff
@@ -0,0 +1,168 @@
+using System.Collections.Generic;
```

Comment:
The module view and presenter is largely copied from BB view and presenter. Refactoring common code didn't make sense because I think the BB explorer will disappear. I opted to keep it for now so I can observe behaviours as I implement the context menu commands.